cg/meta/workflow/taxprofiler.py

```diff
@@ -120,3 +123,22 @@ def config_case(
             header=TaxprofilerSampleSheetEntry.headers(),
         )
         self.write_params_file(case_id=case_id, pipeline_parameters=pipeline_parameters.dict())
+
+    def write_metrics_deliverables(self, case_id: str, dry_run: bool = False) -> None:
```

Comment:
Maybe you could move this implementation to `nf_analysis.py`. The only difference between this one and the rnafusion is `ensure_mandatory_metrics_present`, which could return `None` for taxprofiler until we decide on the metrics. What do you think?

Reply:
Great idea! I agree

Reply:
This is adressed in the latest PR, will resolve the conversation.